src/netmd-interface.ts

```diff
@@ -1030,12 +1069,12 @@ export class MDSession {
             this.hexSessionKey!,
             progressCallback
         );
-        await this.md.cacheTOC();
+        //await this.md.cacheTOC();
         await this.md.setTrackTitle(track, trk.title);
         if (trk.fullWidthTitle) {
             await this.md.setTrackTitle(track, trk.fullWidthTitle, true);
         }
-        await this.md.syncTOC();
+        //await this.md.syncTOC();
```

Comment:
Is there a reason for commenting this 2 `cacheTOC` - `syncTOC` calls?

Now that exceptions are handled into those methods, would it be a problem to leave them uncommented?